python_on_whales/components/pod/cli_wrapper.py

```diff
@@ -0,0 +1,778 @@
+        run(full_cmd)
+
+    ps = list
+
+    def remove(
+        self,
+        x: Union[ValidPod, Sequence[ValidPod]] = (),
+        /,
+        *,
+        all: bool = False,
+        force: bool = False,
+        ignore: bool = False,
+        time: Optional[int] = None,
+    ):
+        """Remove one or more pods.
+
+        Parameters:
+            x: Single pod or list of pods to remove.
+            all: Remove all pods
+            force: Force removal of the pods
+            ignore: Ignore errors when specified pod is missing
+            time: Seconds to wait for pod stop before killing the containers
+
+        # Raises
+            `python_on_whales.exceptions.NoSuchPod` if any of the pods do not
+            exist and `ignore` was not set.
+
+        """
+        pods = to_list(x)
+        if len(pods) == 0 and not all:
+            return
+        full_cmd = self.docker_cmd + ["pod", "rm"]
+        full_cmd.add_flag("--all", all)
+        full_cmd.add_flag("--force", force)
+        full_cmd.add_flag("--ignore", ignore)
+        full_cmd.add_simple_arg("--time", time)
+        full_cmd.extend([str(p) for p in pods])
+        run(full_cmd)
+
+    def restart(
+        self,
+        x: Union[ValidPod, Sequence[ValidPod]] = (),
+        /,
+        *,
+        all: bool = False,
+        latest: bool = False,
+    ) -> None:
+        """Restarts one or more pods
+
+        Parameters:
+            x: One or more pods to restart
+            all: Restart all pods
+            latest: Restart the latest pod
+
+        # Raises
+            `python_on_whales.exceptions.NoSuchPod` if any pods does not exist.
+        """
+        pods = to_list(x)
+        if len(pods) == 0 and not latest and not all:
+            return
+
+        full_cmd = self.docker_cmd + ["pod", "restart"]
+        full_cmd.add_flag("--all", all)
+        full_cmd.add_flag("--latest", latest)
+        full_cmd.extend([str(p) for p in pods])
+
+        run(full_cmd)
+
+    def start(
+        self,
+        x: Union[ValidPod, Sequence[ValidPod]] = (),
+        /,
+        *,
+        all: bool = False,
+        latest: bool = False,
+    ) -> None:
+        """Starts one or more pods
+
+        Parameters:
+            x: One or more pods to start
+            all: Start all pods
+            latest: Start the latest pod
+
+        # Raises
+            `python_on_whales.exceptions.NoSuchPod` if any pods does not exist.
+        """
+        pods = to_list(x)
+        if len(pods) == 0 and not latest and not all:
+            return
+
+        full_cmd = self.docker_cmd + ["pod", "start"]
+        full_cmd.add_flag("--all", all)
+        full_cmd.add_flag("--latest", latest)
+        full_cmd.extend([str(p) for p in pods])
+
+        run(full_cmd)
+
+    def stats(
+        self,
+        x: Union[ValidPod, Sequence[ValidPod]] = (),
+        /,
+        *,
+        all: bool = False,
+        latest: bool = False,
```

Comment:
Same issue here. I wonder if there is a way to say that only one argument must be used. But it's less of an issue since it's not implemented yet.

Reply:
deferred